dtests/src/test/java/io/snappydata/hydra/cdcConnector/cdcConnectorLongRunningHAWithMultiSqlServer.conf

```diff
@@ -15,27 +15,27 @@ INCLUDE $JTESTS/io/snappydata/hydra/cdcConnector/snappyCustomizedHA.inc;
             io.snappydata.hydra.cluster.SnappyPrms-sparkJobClassNames = io.snappydata.app.JavaCdcStreamingApp
-            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=&{coresPerStrmApp} --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
-            io.snappydata.hydra.cluster.SnappyPrms-userAppArgs = "${connectionPropFile1} ${sourceDestFile1} "
+            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=6 --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
+            io.snappydata.hydra.cluster.SnappyPrms-userAppArgs = "${connectionPropFile1} ${sourceDestFile} "
             io.snappydata.hydra.cluster.SnappyPrms-userAppJar = "${pocJar}"
             io.snappydata.hydra.cdcConnector.SnappyCDCPrms-isCDCStream = true
             io.snappydata.hydra.cdcConnector.SnappyCDCPrms-appName= "JavaCdcStreamingApp1"
             threadGroups = snappyTaskThread1
-            startInterval = 15480  //after every 4.30 hours retry submitting the streaming app
-            endInterval   = 15480
+            startInterval = 720  //after every 12 mins retry submitting the streaming app
+            endInterval   = 720
             maxThreads = 1
             ;
 
 TASK        taskClass  = io.snappydata.hydra.cluster.SnappyTest taskMethod  = HydraTask_executeSparkJob
             io.snappydata.hydra.cluster.SnappyPrms-sparkJobClassNames = io.snappydata.app.JavaCdcStreamingApp
-            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=&{coresPerStrmApp} --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
-            io.snappydata.hydra.cluster.SnappyPrms-userAppArgs = "${connectionPropFile2} ${sourceDestFile2} "
+            io.snappydata.hydra.cluster.SnappyPrms-sparkSubmitExtraPrms = " --total-executor-cores=6 --conf spark.scheduler.mode=FAIR --conf spark.sql.defaultSizeInBytes=1000 --conf spark.driver.memory=10g --conf spark.executor.memory=4g --conf snappydata.store.memory-size=12g --conf spark.locality.wait=30 --conf spark.local.dir=/nfs/users/spillai/tmp1 --jars ${connectorJar},${driverJar},${testJar} "
```

Comment:
Formatting changes required for all bts and confs

Reply:
Done